src/resources/Microsoft.Authorization/policyDefinitions/Deploy-ASC-SecurityContacts.json

```
@@ -63,6 +76,10 @@
                 "field": "Microsoft.Security/securityContacts/email",
                 "contains": "[[parameters('emailSecurityContact')]"
               },
+              {
+                "field": "Microsoft.Security/securityContacts/alertNotifications.minimalSeverity",
+                "contains": "[parameters('minimalSeverity')]"
```

```suggestion
                "contains": "[[parameters('minimalSeverity')]"
```
This should be double escaped for ALZ requirements